Activity log: upgrade nudge demo content

simison opened the pull request:
Adds a couple faded demo events together with upgrade banner.

Visible only for wpcom free sites and only with the upgrade nudge banner.

AL with events:

![screen shot 2018-06-29 at 14 56 18](https://user-images.githubusercontent.com/87168/42091104-a3ef2292-7bac-11e8-875a-e6832a6fff90.png)

AL without events:

![screen shot 2018-06-29 at 17 06 21](https://user-images.githubusercontent.com/87168/42096722-cca415f0-7bbe-11e8-81b4-409da0cce0dd.png)


On small screens show only first 2 demo events:

![screen shot 2018-06-29 at 16 19 08](https://user-images.githubusercontent.com/87168/42094374-4559eee0-7bb8-11e8-9aca-9f3b4dcb43ca.png)

I would've preferred not to pass `siteId` here but something inside `ActivityLogItem` really wants it. Will need to work that requirement out or let it be.

keoshi commented:
This is looking great! 🙌

Looking at the code it doesn't seem trivial to do, but would it be possible to have different times for each activity (chronologically ordered, of course) and have the second actor be a visitor, not an admin?

In terms of styling:

- Could we add `margin-bottom: 1px` to `.feature-example` since it's _eating_ the border of the nudge?
- `.feature-example__content` could also use a `margin-right: 1px` to show the right-hand border on the items.
- Should I do these to save you some time?

Everything else looks perfect, thank you!

simison commented:
Thanks @keoshi,  I made similar observations and fixed them. I updated the screenshot.

What do you think, should this demo content be hidden from screen readers or offer some alternative text? (Asked also at p1530270976000202-slack-a11y)

simison commented on client/my-sites/stats/activity-log-example/style.scss:
```diff
@@ -0,0 +1,10 @@
+	.feature-example {
+		// Make overflowing border visible
+		margin-right: -1px;
+		padding-right: 1px;
```
Not a huge fan of this but didn't see any obvious way to fix it directly at `<FeatureExample>`

simison commented:
Updated to show only first 2 events on small screens. 👍

keoshi commented:
Works great! 🙌

simison commented:
I need to hide the example content together with the banner, and there are couple routes it could take but I’m not sure which to follow:

- Build a way to prepend/append elements into the `<Banner>` so that there’s only one place where discarding happens.
    - Kinda heavier approach. 👎 
    - This might ever be the only use case where we need this feature. 👎 

- Query for the same option `<Banner>` uses to hide the demo content.
    - Potentially fragile 👎 
    - Less coding. :thumbsup:

- Remove `<Banner>` wrapper and add our own logic for dismissing the banner. 
    - Easy to extend.  :thumbsup:
    - Might be in future we don't have a thing here that looks like a "banner" but something more elaborate? Ping @keoshi
